app/App/Http/ViewModels/ProfileViewModel.php

```diff
@@ -0,0 +1,8 @@
+<?php
+namespace App\Http\ViewModels;
+
+use Spatie\ViewModels\ViewModel;
+
+class ProfileViewModel extends ViewModel
```

Comment:
I believe you forget the constructor here. According to the controller, it should take a User.

Simply setting the user as a public property on the view model will be enough for it to end up as a variable in the view, or you could make `ProfileViewModel::$user` private, and expose is via a public `ProfileViewModel::user()` method.

Reply:
Do you prefer to use a getter or expose the properties in public?

Reply:
I prefer the getter approach 😄

Reply:
Me too ;)